.github/scripts/generate_binary_build_matrix.py

```diff
@@ -56,8 +56,8 @@ def arch_type(arch_version: str) -> str:
 WHEEL_CONTAINER_IMAGES = {
     "11.8": "pytorch/manylinux-builder:cuda11.8-59a2f92aa12c3c0cb11622b05fe77de8312f6d00",
     "12.1": "pytorch/manylinux-builder:cuda12.1-59a2f92aa12c3c0cb11622b05fe77de8312f6d00",
-    "5.5": "pytorch/manylinux-builder:rocm5.5-01ce69ff0d18320ab36787e288436586ed278692",
-    "5.6": "pytorch/manylinux-builder:rocm5.6-01ce69ff0d18320ab36787e288436586ed278692",
+    "5.5": "pytorch/manylinux-builder:rocm5.5-77901321d9387eeb68ca9e2ddab7562883562cd1",
+    "5.6": "pytorch/manylinux-builder:rocm5.6-77901321d9387eeb68ca9e2ddab7562883562cd1",
```

Comment:
We already have `-2.1` suffix, why not use it?
```suggestion
    "5.5": "pytorch/manylinux-builder:rocm5.5-2.1",
    "5.6": "pytorch/manylinux-builder:rocm5.6-2.1",
```

Reply:
This is much better indeed.  I will follow up with a similar change to pin trunk to `-main`.  On the other hand, https://registry.hub.docker.com/r/pytorch/libtorch-cxx11-builder and https://registry.hub.docker.com/r/pytorch/conda-builder images don't have `-2.1` tags.  It looks like something is missing there, I will check that before switching the rest of the images here to `-2.1`.